model-optimizer/unit_tests/mo/utils/versions_checker_test.py

```diff
@@ -86,6 +87,47 @@ def test_append_version_list(self):
         for i, v in enumerate(req_list):
             self.assertEqual(v, ref_list[i])
 
+    def test_append_version_list_sys_neg_1(self):
+        v1 = "mxnet>=1.7.0 ; sys_platform != 'win32'"
+        req_list = list()
```

Comment:
Why do you create list object here using `list()` and in lines 102, 119, 127 use `[]`? Let's have consistent approach. Lets use `[]` everywhere.

Reply:
Ok, aligned.